addons/diagnostic/XEH_preInit.sqf

```diff
@@ -18,7 +20,7 @@ GVAR(projectileTrackedUnits) = [];
 
 ADDON = true;
 
-if (getMissionConfigValue ["EnableTargetDebug", 0] == 1 || {getNumber (configFile >> "EnableTargetDebug") == 1}) then {
+if (getMissionConfigValue ["EnableTargetDebug", 0] == 1 || {getNumber (configFile >> "EnableTargetDebug") == 1} || {GVAR(forceTargetDebug)}) then {
```

Comment:
this happens before settings are initialized

Reply:
`#include "initSettings.sqf"`
Is called before, to init the settings.
I have tested solo and multi with and without the option on mission without the mission's config selected, that works great

Reply:
The global variable is definitely not synchronized from the server at preInit. This is a global setting. Try on a server as dedicated client.

Reply:
I will test more with a dedicated client and a dedicated server
I could add a waitUntil in the condition, not executed if mission is set to true, however, it will wait the synchronizatio... still have to check what variable I can watch to know when it's done

Thx for the explanation.

Reply:
Oki added a commit, that wait the event CBA_settingsInitialized event